### PR TITLE
GRW-107 / Feat / Danish language picker

### PR DESCRIPTION
**Pull request by styrelius**
## What?

Language picker that works for `/dk` and `/dk-en`! 🇩🇰 

- New structure for the `LanguagePicker` with a `languagePickerData` file where the locale specific data is.
- Styling fixes since some of the styles broke with the new structure 😔 
- Tests for the Danish language picker in `LanguagePicker.test.tsx` and new snapshots since the existing ones became obsolete/broken. Also some changes to the tests for the Norwegian and Swedish language picker.

**Bonus:** `LanguagePicker` added to `/offer` page for consistency and maybe you realise at this step that you're not as fluent in Swedish/Norwegian/Danish as you though and need this in English??! The way we currently have it is you have to start the flow over (if you're not smart and somehow figure out that you can directly change the url).

## Why?

(Maybe we should remove this section of the template LOL, it's usually pretty self-explanatory)

Because of Denmark! And regarding the refactoring the reason is of course the same as with the `Landing` page - it's easier to scale with this kind of data structure.

_Referenced ticket(s): [GRW-107]_

## Screen recordings

**Denmark**

2021-03-22-language-picker-DK

**Sweden, /offer page**

2021-03-22-language-picker-offer-SE

[GRW-107]: https://hedvig.atlassian.net/browse/GRW-107

### CODE REVIEW

File: src/client/pages/OfferNew/index.tsx [hunk 1]
```diff
@@ -96,7 +97,9 @@ export const OfferNew: React.FC = () => {
     <Page>
       <SessionTokenGuard>
         {![Variation.IOS, Variation.ANDROID].includes(variation!) && (
-          <TopBar isTransparent />
+          <TopBar isTransparent>
+            <LanguagePicker path="/new-member/offer" />
+          </TopBar>
```
Comment (gustaveen) [inline src/client/pages/OfferNew/index.tsx:102]: 👌

File: src/client/pages/Embark/LanguagePicker.tsx [hunk 2]
```diff
@@ -40,13 +44,15 @@ const Divider = styled.span`
 `
 
 const Dropdown = styled.div`
+  height: 80px;
```
Comment (gustaveen) [inline src/client/pages/Embark/LanguagePicker.tsx:47]: Any particular reason we want to restrict the height? 🙃  

https://user-images.githubusercontent.com/6661511/112007783-3a1d9480-8b25-11eb-8073-b6fca036569c.mov

**Review by gustaveen — COMMENTED** [review]
Awesome! 🌏. 🙌  Just one question

File: src/client/pages/Embark/LanguagePicker.tsx (hunk 2 above)
Comment (styrelius) [reply src/client/pages/Embark/LanguagePicker.tsx:47]: Nope, just accidentally left it there after trying to fix styling issues. Also that `Dropdown` component of course doesn't need `display: flex` so I'll remove that as well 👍 🙏

**Review by gustaveen — APPROVED** [review]
🚀

**Pull request merged**